src/Schema/Events/CollectorProfile.ts

```diff
@@ -0,0 +1,31 @@
+import { ContextModule } from "../Values/ContextModule"
+import { OwnerType } from "../Values/OwnerType"
+import { ActionType } from "."
+import { Platform } from "./MyCollection"
+
+/**
+ * Schema describing 'Edited Collector Profile' events
+ * @packageDocumentation
+ */
+
+/**
+ * A user taps or clicks on "Save" when editing their collector profile info.
+ *
+ * This schema describes events sent to Segment from [[editedUserProfile]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "editedUserProfile",
+ *    context_screen: "collectorProfile",
+ *    context_screen_owner_type: "editProfile",
+ *    platform: "mobile"
+ *  }
+ * ```
+ */
+export interface EditedUserProfile {
+  action: ActionType.editedUserProfile
+  context_screen: ContextModule.collectorProfile
+  context_screen_owner_type: OwnerType.editProfile
```

Comment:
**Suggestion:** Since we are using the same event for both mobile and web, and the keyword `screen` indicates that it's from mobile, should we change these to `context_module...` instead?

Reply:
Both sound OK to me. We can use the context_module is you think that this makes more sense.

Reply:
@audrey-thenot what do you think?

Reply:
I believe it would be okay to have it as is now, I just wanted to check with Audrey about it :) will merge now!

Reply:
Sure - please feel free to reach out to Audrey personally if blocked